Data_Engine/Create/Domain.cs

```diff
@@ -39,14 +39,24 @@ public static partial class Create
         /***************************************************/
 
         [Description("Creates a numerical domain around the two values.")]
-        [Input("min", "The lowest value in the domain, will be set as max if larger than max.")]
-        [Input("max", "The highest value in the domain, will be set as min if larger than min.")]
+        [Input("min", "The lowest value in the domain, will be set as max if larger than max. Defaults to -infinite (double.MinValue).")]
+        [Input("max", "The highest value in the domain, will be set as min if larger than min. Defaults to +infinite (double.MaxValue).")]
         [Output("domain", "A numerical domain with its extreme values at min and max.")]
-        public static Domain Domain(double min, double max)
+        public static Domain Domain(double min = double.MinValue, double max = double.MaxValue)
         {
             return new Domain(Math.Min(min, max), Math.Max(min, max));
         }
 
+        [Description("Creates a numerical domain around the two values.")]
+        [Input("min", "The lowest value in the domain, will be set as max if larger than max. Defaults to -infinite (double.MinValue).")]
+        [Input("max", "The highest value in the domain, will be set as min if larger than min. Defaults to +infinite (double.MaxValue).")]
+        [Input("extremesInclusion", "Whether the Min and Max extremes are included in the Domain or not. `IncludeBoth` by default.")]
+        [Output("domain", "A numerical domain with its extreme values at min and max.")]
+        public static Domain Domain(double min = double.MinValue, double max = double.MaxValue, BH.oM.Data.DomainExtremesInclusion extremesInclusion = BH.oM.Data.DomainExtremesInclusion.IncludeBoth)
```

Comment:
Given that both have default options across the board, what is the benefit of both create methods? Seems like it'd be better to just have one?

Reply:
That and the fact that the system will not be able to decide which method to call for less than three arguments.
Agreed that the first one should not exist.

Reply:
@adecler , @FraserGreenroyd thanks and agreed - removed and added versioning, plus test script.